pkgs/games/steam/fhsenv.nix

```diff
@@ -195,6 +195,8 @@ in buildFHSUserEnv rec {
     SDL2_ttf
     SDL2_mixer
     libappindicator-gtk2
+    libindicator-gtk2
+    libdbusmenu-gtk2
```

Comment:
Very minor nit that I'm not sure really matters, but maybe these would be better being swapped (so they're alphabetical relative to the other GTK 2 libraries)?

Reply:
Thanks for doing that for me @SuperSandro2000 !